pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Subscription.java

```diff
@@ -53,7 +53,7 @@ default void removeConsumer(Consumer consumer) throws BrokerServiceException {
 
     Dispatcher getDispatcher();
 
-    long getNumberOfEntriesInBacklog();
+    long getNumberOfEntriesInBacklog(boolean isPreciseBacklog);
```

Comment:
just a suggestion for future tasks:

if we are changing a method to add another flag, it would be better to add an overloaded method rather than directly updating the original method. It will reduce the number of files to be changed.

```
default long getNumberOfEntriesInBacklog() {
    return getNumberOfEntriesInBacklog(false);
}

long getNumberOfEntriesInBacklog(boolean isPreciseBacklog);
```

Reply:
Got it.